generator/uscore/uscore_generator.rb

```diff
@@ -703,31 +742,38 @@ def get_first_search_with_fixed_values(sequence, search_parameters, save_resourc
         find_two_values = get_multiple_or_params(sequence).include? search_param[:name]
         values_variable_name = "#{search_param[:name].tr('-', '_')}_val"
         %(
-          @#{sequence[:resource].underscore}_ary = []
+          @#{sequence[:resource].underscore}_ary = {} 
+          @resources_found = false
           #{'values_found = 0' if find_two_values}
           #{values_variable_name} = [#{search_param[:values].map { |val| "'#{val}'" }.join(', ')}]
-          #{values_variable_name}.each do |val|
-            search_params = { 'patient': @instance.patient_id, '#{search_param[:name]}': val }
-            reply = get_resource_by_params(versioned_resource_class('#{sequence[:resource]}'), search_params)
-            assert_response_ok(reply)
-            assert_bundle_response(reply)
+          could_not_resolve_all = []
```

Comment:
We don't need the resolved_one logic in any of the first searches because we're not resolving the values from the resources. They're all either just patient or patient + some fixed value.